### ISSUE TITLE
Circular reference in schema leads to Could not resolve reference: undefined undefined

### ISSUE THREAD

**Issue by TimoKunze** [swagger-api/swagger-ui#5726]
### Q&A (please complete the following information)
 - OS: Alpine Linux
 - Browser: Vivaldi
 - Version: 2.9
 - Method of installation: docker
 - Swagger-UI version: actually unknown, should be latest
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
Example Swagger/OpenAPI definition:
```yaml
openapi: "3.0.0"
info:
  title: "Test API For Self Reference"
  version: "1.0"


paths:
  /greeting:
    get:
      tags:
        - Client
      description: 'Get the current client session identifier.'
      responses:
        200: 
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Person'
        
components:
  schemas:
    Person:
      type: object
      required:
        - name
        - age
      properties:
        name:
         type: string
         example: 'John Doe'
        age:
          type: integer
          example: 42
        parent:
          $ref: '#/components/schemas/Person'
        brother:
          $ref: '#/components/schemas/Person'
```

Swagger-UI configuration options:
Where do I find it?

### Describe the bug you're encountering
When expanding the `Person` schema, Swagger-UI displays this error on top of the page:
Resolver error at components.schemas.Person.properties.parent.$ref
Could not resolve reference: undefined undefined

### To reproduce...

1. Scroll down to 'schemas'
2. Click on 'Person' to expand it
3. Scroll up to the top of the page

### Expected behavior
No error should occur.

**Comment by TimoKunze** [comment]
Hmm, I kind of fixed it by entering the absolute path (instead of relative one) to the yaml file in the Explore text field. I'm not sure whether this is the way it is meant to be...

**Comment by webron** [comment]
Are you sure you're using the latest version? Can't seem to reproduce it on editor.swagger.io.

**Comment by hnthu** [comment]
I have met the same issue for using 3.25.2.
I have a quick check and figure out the issue happened from 3.22.3
Have check on editor.swagger.io

it may empty instead of throw warning on the top of the page

![image](https://user-images.githubusercontent.com/25275587/81548766-ffe15380-93a7-11ea-9d0f-d0326315c4b6.png)

**Comment by bligny** [comment]
The issue is still there (5 months later).
Any fix foreseen ?

**Comment by cgockeln42** [comment]
Is here any work done so far? Highly interested in a fix

**Comment by meredrica** [comment]
we have the same issue. any help?

**Comment by Messhias** [comment]
2 years passed, no one will look at it anymore.

**Comment by meredrica** [comment]
Yep @Messhias we are already looking alternatives. Rapi docs looks promising

**Comment by Messhias** [comment]
> Yep @Messhias we are already looking alternatives. Rapi docs looks promising

Hi @meredrica can you shade the link of it? 

Thanks,

**Comment by wangdashuaihenshuai** [comment]
why swagger ui support circle referrece so diffcult.
